File: cpp/bmi.cpp
/*
 * bmi.cpp
 */


#include <iostream>
using namespace std;

int main()
{
    float waga;
    float wzrost;
    float bmi;
    
    cout << "Podaj swoja wage: ";
    cin >> waga; 
    cout << "Podaj swoj wzrost w centymetrach: ";
    cin >> wzrost;
    bmi = (waga /(wzrost * wzrost)* 10000);
    cout << "Twoje bmi wynosi: " << bmi <<endl;
    cout << "Typ wagi: ";
    
    if(bmi < 18.5)
        cout << "Niedowaga" << endl;
    if(bmi >= 18.5 && bmi < 24.9)
        cout << "Norma" << endl;
    if(bmi >=25 && bmi < 30)
        cout << "Nadwaga" << endl;
    if(bmi >= 30)
        cout << "Otyłość" << endl;
   
    return 0;
}

File: python/bmi.py
#!/usr/bin/env python
# -*- coding: utf-8 -*-
#
#  bmi.py
#


def main(args):
    
    wzrost = float(input('Podaj swoj wzrost [cm]: '))
    waga= int(input('Podaj swoja wage [kg]: '))
    bmi = round(waga/(wzrost*wzrost)*10000)

    if bmi <= 18.5:
        print('BMI: ', bmi,'niedowaga')

    elif bmi > 18.5 and bmi < 25:
        print('BMI:', bmi,'norma')

    elif bmi > 25 and bmi < 30:
        print('BMI:', bmi,'nadwaga')

    elif bmi > 30:
        print('BMI:', bmi,'otylosc')
    
    
    return 0

if __name__ == '__main__':
    import sys
    sys.exit(main(sys.argv))
